bcda/main.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/CMSgov/bcda-app/bcda/monitoring"
 	"github.com/CMSgov/bcda-app/bcda/servicemux"
 
-	"github.com/bgentry/que-go"
+	que "github.com/bgentry/que-go"
```

Comment:
You need to fix your IDE so that it stops putting in these duplicate aliases. What's happening here is that the package is name que, but the file is not. That's perfectly legal in go.

Reply:
hmm yeah, i guess the point is is that when you use it, you call it "que".  so you can tell how it will be used from the import statement.  i'll find where the setting is and switch it off

Reply:
using goimports on save instead of goreturns solved it!